.stickler.yml

```diff
@@ -0,0 +1,6 @@
+linters:
+  csslint: {  }
+files:
+  ignore:
+    - 'bower_components/*'
+    - 'node_modules/*'
```

Comment:
- See the comment above, as the way your stickler file is setup is not listening to the rules provided by the eslintrc file.